jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererUserGroupSettings.kt

```diff
@@ -79,8 +79,10 @@ class CodeWhispererUserGroupSettings : PersistentStateComponent<CodeWhispererUse
     @VisibleForTesting
     fun determineUserGroup(): CodeWhispererUserGroup {
         val randomNum = Math.random()
-        val group = if (randomNum < 1 / 2.0) {
+        val group = if (randomNum < 1 / 3.0) {
```

Comment:
are we aligned 1/3 users will be assigned to each group?

Reply:
yes we aligned on the user group deviding